README.rst

```diff
@@ -0,0 +1,44 @@
+json-python-coverage
+====================
```

Comment:
The first heading in the document translates to H1 in HTML and it should be unique. There is no fixed order of headings specified in RST (it's determined solely by their order in the document). In our projects we usually follow this convention:

```rst
===========
 Header H1
===========

-----------
 Header H2
-----------

Header H3
=========

Header H4
---------
```

If there is need another levels (which is quite rare), we usually use `^^^` underline to mark header H5 and others at the developer discretion.

Reply:
Fixed